src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,78 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagesQnt
+    = total % perPage === 0 ? total / perPage : Math.ceil(total / perPage);
```

Comment:
Why just don't: ? 
The condition looks useless for me

And try to make full and meaningful names
```suggestion
  const pagesCount = Math.ceil(total / perPage);
```